app/scripts/lib/metrics.js

```diff
@@ -364,6 +367,27 @@ define(function (require, exports, module) {
       // The idenifier is kept to avoid updating all metrics queries.
       return 'screen.' + viewName;
     },
+    /**
+     * Log when an experiment is shown to the user
+     *
+     * @param {String} choice - type of experiment
+     * @param {String} group - the experiment group (treatment or control)
+     */
+
+    logExperiment: function (choice, group) {
+      choice = choice || UNKNOWN_EXPERIMENT_CHOICE;
```

Comment:
just do a check here if `choice` or `group` are not given to you then just ignore logging (a bit different from marketing impressions)